examples/basic/terraform.tfvars

```diff
@@ -0,0 +1,3 @@
+namespace = "eg"
+name = "test"
+stage = "example"
```

Comment:
let's be consistent, either add all vars to `terraform.tfvars` or add default values to `variables.tf`
(we have default for `name` now, but not for the rest of the vars)

Reply:
I don't quite follow. Can you elaborate?

Reply:
the example has some vars defined in `terraform.tfvars` and some in `variables.tf` defaults.
let's define them in one place - easier to read and understand

Reply:
@aknysh 
in my opinion (as non devops engineer, but like java monkey coder) it would be better init and define variables in `variables.tf`.
it much more understandable even to me)

Reply:
@osterman  what's your opinion on that?
`variables.tf` has some vars defined in the file, but some in `terraform.tfvars`.
For consistency, what's the best way to do it for **TF examples**?
Define all vars in `variables.tf`, or remove al defaults from `variables.tf` and define all of them in `terraform.tfvars`?

Reply:
Okay - for *examples* I agree. Let's remove defaults from `variables.tf` and set all defaults in `terraform.tfvars` files

Reply:
@solairerove  please remove all defaults from `variables.tf` and add them to `terraform.tfvars`

Reply:
done